hyperbook/book/faq.md

```diff
@@ -0,0 +1,127 @@
+---
+name: F.A.Q.
+index: 4
+---
+
+# Häufig gestellte Fragen (F.A.Q.)
+Hier sammeln wir einige häufig gestellten Fragen, 
+die uns erreichen.
+Wir versuchen diese stets auktuell
+und ausführlich zu halten.
+
+Wenn du dennoch nicht fündig wirst,
+dann frage uns gerne auf unseren [Discord](https://discord.gg/jhyF7EU),
+oder schreibe uns eine [E-Mail](mailto:tech@software-challenge.de).
+
+## Technisch
+
+### Mein Spieler überschreitet die Zwei-Sekunden-Begrenzung, was kann ich tun?
+
+Eine grundlegende Regelung bei der Software-Challenge ist, 
+dass ein jeder Spieler maximal zwei Sekunden Zeit hat, 
+eine Antwort an den Spielleiter zu senden.
+
+Die Zeit wird von Erhalt der Nachricht des Spielleiters bis zum Erreichen der Antwort des Spielers gezählt.
+
+#### Allgemein
+Es kann nun versucht werden, 
+den Algorithmus zu optimieren, 
+welcher genutzt wird, 
+um den Zug zu berechnen. Beachtet werden kann dabei, 
```

Comment:
solche kurzen sätze sind teilweise in einer Zeile verständlicher, aber beim punkt definitiv einen linebreak setzen